subprojects/testing-jvm/src/integTest/groovy/org/gradle/testing/fixture/TestNGCoverage.groovy

```diff
@@ -21,7 +21,7 @@ import org.gradle.integtests.fixtures.RepoScriptBlockUtil
 import org.gradle.internal.jvm.Jvm
 
 class TestNGCoverage {
-    final static String NEWEST = Jvm.current().javaVersion.java7Compatible ? '6.14.3' : '6.8.7'
+    final static String NEWEST = Jvm.current().javaVersion.java7Compatible ? '7.4.0' : '6.8.7'
```

Comment:
I think this is always true now, Jvm.current would mean that we're running the Gradle test worker with Java 7, which we can't do anymore.

Reply:
OK, so I can remove the ternary expression and always use `7.4.0`?

Also I pinged @marcphilipp to see if there's any reason why we previously were not testing with TestNG v7.

Reply:
Changed to `7.4.0` with no ternary expression.